docs/dev/tutorials/cross-chain-tutorial.md

```diff
@@ -1,64 +1,115 @@
 # Cross-chain governance
 
-This tutorial serves as an example of how to implement L1 to L2 contract interaction. The following functionality is implemented in this tutorial:
+This tutorial shows you how to implement communication between L1 and L2 with the following example:
 
-- A "counter" smart contract is deployed on zkSync, which stores a number that can be incremented by calling the `increment` method.
-- A "governance" smart contract is deployed on layer 1, which has the privilege to increment the counter on zkSync.
+- A **governance** smart contract is deployed on layer 1. This contract has a function that can request a transaction on zkSync layer 2.
+- A **counter** smart contract is deployed on zkSync layer 2. The contract stores a number that can be incremented by calling the `increment` method. The goverance contract on layer 1 calls this function.
 
+## Prerequisites
 
+- You are already familiar with deploying smart contracts on zkSync. If not, please refer to the first section of the [quickstart tutorial](../building-on-zksync/hello-world.md).
+- You already have some experience working with Ethereum.
+- You have a web3 wallet app which holds some Goerli test ETH and some zkSync test ETH.
 
+:::warning
```

Comment:
I would use the phrase:

"The `yarn` instructions have been written to be used with Node 14."